tests/backup/backup_helper.go

```diff
@@ -80,6 +80,8 @@ const (
 	jobDeleteRetryTime                        = 10 * time.Second
 	podStatusTimeOut                          = 20 * time.Minute
 	podStatusRetryTime                        = 30 * time.Second
+	licenseCountUpdateTimeout                 = 15 * time.Minute
+	licenseCountUpdateRetryTime               = 1 * time.Minute
```

Comment:
Why don't we make this 30 seconds?

Reply:
license count update takes more than 30 sec to get updated